nifi-extension-bundles/nifi-py4j-bundle/nifi-py4j-integration-tests/src/test/java/org.apache.nifi.py4j/PythonControllerInteractionIT.java

```diff
@@ -595,6 +595,21 @@ public void testCreateFlowFile() throws IOException {
                 multiLineContent.getBytes(StandardCharsets.UTF_8));
     }
 
+    @Test
+    public void testCreateNothing() throws IOException {
+        /* Test the use-case where the source processor returns with None.
+           (The case must not result in Java NullPointerException.) */
+
+        final String processorName = "CreateNothing";
+        final String relationshipSuccess = "success";
```

Comment:
These variables are just static values that are used once. Would recommend inlining the values rather than creating variables.